projects/museumpassmusees/reference/partner-api.json

```diff
@@ -430,9 +430,188 @@
+        "security": [
+          {
+            "CLIENT_ACCESS_TOKEN": []
+          }
+        ],
+        "parameters": [],
+        "tags": [
+          "External cards"
+        ]
+      },
+      "parameters": [
+        {
+          "schema": {
+            "type": "string",
+            "enum": [
+              "uitpas",
+              "akaart",
+              "article27"
+            ]
+          },
+          "name": "type",
+          "in": "path",
+          "required": true,
+          "description": "The type of external card"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "identification",
+          "in": "path",
+          "required": true,
+          "description": "The identification (e.g. card number) of the external card"
+        }
+      ]
+    },
+    "/external-cards/{type}/{identification}/visits": {
+      "post": {
+        "summary": "Register a museum visit for an external card",
+        "operationId": "post-external-cards-type-identification-visits",
+        "responses": {
+          "201": {
+            "description": "External card visit registered."
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Possible error types:\n\n* https://api.publiq.be/probs/url/not-found",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          },
+          "503": {
+            "description": "Service Unavailable. Possible error types:\n\n* https://api.publiq.be/probs/museumpass/external-service-unavailable\n\nThe `Retry-After` HTTP Header might include a hint when a retry of this request can be attempted.",
+            "content": {
+              "application/problem+json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
+          }
+        },
+        "description": "Registers a visit of an axternal card at your museum (determined by the client access token used).\n\nTo check if this card is allowed to visit, look up their information using [`GET /external-cards/{type}/{identification}`](./partner-api.json/paths/~1get-external-cards-type-identification~1{type}~1{identification}/get). The response to that request will indicate if the passholder may visit your museum.\n\nIf the external card may not visit your museum and you still try to register a visit for them, the server will return a `403 Forbidden` error response.\n\n## Permissions\n\nThe caller of this request must have the `external card visit registrar` role.",
```

Comment:
> look up their information using [`GET /external-cards/{type}/{identification}`](./partner-api.json/paths/~1get-external-cards-type-identification~1{type}~1{identification}/get)

When I try the link on https://docs.publiq.be/docs/museumpassmusees/branches/museumpassmusees%2FPUBMPM-932-partner-api/2d1c386cb2642-register-a-museum-visit-for-an-external-card it does not work. But I don't see the issue immediately either. Maybe try with an absolute path? (So prefixing it with `/reference/` instead of `./`)

Reply:
@erwin1 I tried it again after your change to prefix it with `/reference` but that didn't fix it either. I noticed that the actual path was incorrect so I fixed it in [82f233d](https://github.com/cultuurnet/apidocs/pull/39/commits/82f233d66af72a3df39283a98932f5ea6519d636).

One way to easily get the correct URL is by right-clicking the endpoint in Studio and clicking "Copy relative path". Then you can paste wherever you need it, and add `/get` or `/post` or ... to the URL depending on the method you want to link to.